pkgs/top-level/all-packages.nix

```diff
@@ -15100,7 +15100,7 @@ with pkgs;
 
   bump = callPackage ../development/tools/github/bump { };
 
-  libbpf = callPackage ../os-specific/linux/libbpf { };
+  libbpf = linuxPackages.libbpf;
```

Comment:
I am unsure that libbpf should even be a top-level package at all, but I kept it for consistency.

Reply:
Has it binary tools?

Reply:
No, but as @vcunat pointed out we do not want to rebuild all packages that depend on libbpf for each kernel upgrade. libbpf itself only uses stable kernel ABI and should work regardless of the kernel version (as long as features the user tries to use are available)

(I've moved bcc out of linuxPackages because it was absolutely horrible to use, I'd rather these tools not go back in)